apps/web/e2e/record_maintenance.py

```
@@ -0,0 +1,52 @@
+Env:
+    BASE_URL  default http://localhost:3000
+    OUT_DIR   default /tmp/maint-rec   (gets a .webm video + per-page PNGs)
+"""
+
+import os
+import pathlib
+
+from playwright.sync_api import sync_playwright
+
+BASE = os.environ.get("BASE_URL", "http://localhost:3000")
+OUT = pathlib.Path(os.environ.get("OUT_DIR", "/tmp/maint-rec"))
```

Ruff (0.15.17)

[error] Probable insecure usage of temporary file or directory: "/tmp/maint-rec"

(S108)